### PR TITLE
[Merged by Bors] - style: fix references to DenselyOrdered

### PR DESCRIPTION
**Pull request by sven-manthe**
Rename occurrences of "densely_ordered" to "denselyOrdered"

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Data/PSigma/Order.lean [hunk 1]
```diff
@@ -153,15 +153,15 @@ instance denselyOrdered_of_noMaxOrder [Preorder ι] [∀ i, Preorder (α i)]
       exact ⟨⟨i, c⟩, right _ ha, right _ hb⟩⟩
 #align psigma.lex.densely_ordered_of_no_max_order PSigma.Lex.denselyOrdered_of_noMaxOrder
 
-instance densely_ordered_of_noMinOrder [Preorder ι] [∀ i, Preorder (α i)]
+instance denselyOrdered_of_noMinOrder [Preorder ι] [∀ i, Preorder (α i)]
```
Comment (grunweg) [inline Mathlib/Data/PSigma/Order.lean:156]: I believe the naming convention is that manual instance names should start with "inst", i.e.
```suggestion
instance instDenselyOrdered_of_noMinOrder [Preorder ι] [∀ i, Preorder (α i)]
```
(I'm not 100% sure on the casing; you may want to compare with other names/wait for another pair of eyes.)

File: Mathlib/Order/CountableDenseLinearOrder.lean [hunk 2]
```diff
@@ -61,7 +61,7 @@ theorem exists_between_finsets {α : Type*} [LinearOrder α] [DenselyOrdered α]
         Exists.elim
         (exists_lt (Finset.min' hi nhi)) fun m hm ↦
         ⟨m, fun x hx ↦ (nlo ⟨x, hx⟩).elim, fun y hy ↦ lt_of_lt_of_le hm (Finset.min'_le hi y hy)⟩
-    else-- both sets are empty, use nonempty
+    else-- both sets are empty, use `Nonempty`
```
Comment (grunweg) [inline Mathlib/Order/CountableDenseLinearOrder.lean:64]: While you're at it:
```suggestion
    else -- both sets are empty, use `Nonempty`

File: Mathlib/Order/Cover.lean [hunk 3]
```diff
@@ -245,10 +245,10 @@ theorem not_covBy [DenselyOrdered α] : ¬a ⋖ b := fun h =>
   h.2 hc.1 hc.2
 #align not_covby not_covBy
 
-theorem densely_ordered_iff_forall_not_covBy : DenselyOrdered α ↔ ∀ a b : α, ¬a ⋖ b :=
```
Comment (grunweg) [inline Mathlib/Order/Cover.lean:248]: You're renaming a theorem: can you add a deprecation warning and an allias, with a current date?
```suggestion
@[deprecated] alias densely_ordered_iff_forall_not_covBy := denselyOrdered_iff_forall_not_covBy -- 2024-04-04

**Review by grunweg — APPROVED** [review]
Thank you for doing this; good find! I have two follow-up suggestions which should be quick to address.

File: Mathlib/Data/PSigma/Order.lean (hunk 1 above)
Comment (sven-manthe) [reply Mathlib/Data/PSigma/Order.lean:156]: If I do this, there will be many more examples of such instances in this file alone

Comment (grunweg) [reply Mathlib/Data/PSigma/Order.lean:156]: Sure; mathlib is not always consistent (and I think a fix for that would be welcome - to be sure, I'd ask on zulip first).

I'm not asking you to change any other instances! But when you're renaming this one, I'd rather rename it to a "right" than a "less wrong" name.

Comment (sven-manthe) [reply Mathlib/Data/PSigma/Order.lean:156]: Personally, I would find the resulting inconsistency with e.g. denselyOrdered_of_noMaxOrder even more confusing

**Comment by ocfnash** [comment]
Thanks!

bors merge

**Pull request closed without merging**